app/ui/legacy/src/main/java/com/fsck/k9/view/ViewUtils.kt

```diff
@@ -0,0 +1,36 @@
+package com.fsck.k9.view
+
+import android.view.View
+import android.view.animation.AccelerateDecelerateInterpolator
+
+private const val FADE_DURATION = 300L
+private const val ALPHA_VISIBLE = 1.0f
+private const val ALPHA_GONE = 0.0f
+
+internal fun View.makeVisible() {
+    visibility = View.VISIBLE
+}
+
+internal fun View.makeGone() {
+    visibility = View.GONE
+}
```

Comment:
there is a android core library which include those as `isGone` and `isVisible`
https://android.github.io/android-ktx/core-ktx/androidx.view/android.view.-view/is-gone.html